google-cloud-bigquerydatatransfer/README.md

```diff
@@ -114,7 +111,6 @@ Apache 2.0 - See [LICENSE] for more information.
 [LICENSE]: https://github.com/GoogleCloudPlatform/google-cloud-java/blob/master/LICENSE
 [TESTING]: https://github.com/GoogleCloudPlatform/google-cloud-java/blob/master/TESTING.md#testing-code-that-uses-bigquerydatatransfer
 
-[cloud-bigquerydatatransfer]: https://cloud.google.com/bigquerydatatransfer/
-[cloud-bigquerydatatransfer-quickstart]: https://cloud.google.com/bigquerydatatransfer/quickstart-console#before-you-begin
-[bigquerydatatransfer-product-docs]: https://cloud.google.com/bigquerydatatransfer/docs/
+[cloud-bigquerydatatransfer]: https://cloud.google.com/bigquery/docs/reference/datatransfer/rest/
```

Comment:
This seems like a better URL: https://cloud.google.com/bigquery/docs/transfer-service-overview

(And it seems fine to use it for the docs URL too)

Reply:
Done.